src/resources/Resources/tests/Resources.spec.ts

```diff
@@ -25,7 +25,7 @@ describe('Resources', () => {
             expect(api.get).toHaveBeenCalledWith(`${Resources.baseUrl}/SOURCE`);
         });
 
-        it('should use the passed parameters as its query paramaters', () => {
```

Comment:
![image](https://github.com/coveo/platform-client/assets/45688129/7fed7f02-8563-4a9a-829b-9bcb4bc06bc9)

The ParaMaters,  new pixar movie coming out in 2024